src/config.xml

```diff
@@ -3545,33 +3545,56 @@ to be found in the default search path.
 ]]>
       </docs>
     </option>
-    <option type='string' id='DOT_FONTNAME' format='string' defval='Helvetica' depends='HAVE_DOT'>
+    <option type='string' id='DOT_GRAPH_ATTR' format='string' defval='fontname=Helvetica, fontsize=10' depends='HAVE_DOT'>
       <docs>
-<![CDATA[
- When you want a differently looking font in the dot files that doxygen generates
- you can specify the font name
- using \c DOT_FONTNAME. You need to make sure dot is able to find the font,
- which can be done by putting it in a standard location or by setting the
- \c DOTFONTPATH environment variable or by setting \ref cfg_dot_fontpath "DOT_FONTPATH" to the
- directory containing the font.
-]]>
+        <![CDATA[
+        <a href=https://graphviz.org/doc/info/attrs.html>Node, Edge and Graph Attributes specification</a>
+        ]]>
       </docs>
     </option>
-    <option type='int' id='DOT_FONTSIZE' minval='4' maxval='24' defval='10' depends='HAVE_DOT'>
+    <option type='string' id='DOT_COMMON_ATTR' format='string' depends='HAVE_DOT'
+      defval='fontname=Helvetica, fontsize=10'>
       <docs>
-<![CDATA[
- The \c DOT_FONTSIZE tag can be used to set the size (in points) of the font of dot graphs.
-]]>
+        <![CDATA[
+        DOT_COMMON_ATTR is common attributes for nodes and edges.
+        When you want a differently looking font in the dot files that doxygen generates
+        you can specify fontname attribute.
+        <a href=https://graphviz.org/doc/info/attrs.html>Node, Edge and Graph Attributes specification</a>
+        You need to make sure dot is able to find the font,
+        which can be done by putting it in a standard location
+        or by setting the \c DOTFONTPATH environment variable or by
+        setting \ref cfg_dot_fontpath "DOT_FONTPATH" to the
+        directory containing the font.
+        Default graphviz fontsize is 14.
+        ]]>
+      </docs>
+    </option>
+    <option type='string' id='DOT_EDGE_ATTR' format='string' depends='HAVE_DOT'
+      defval='labelfontsize=10'>
+      <docs>
+        <![CDATA[
+        DOT_EDGE_ATTR is concatenated with DOT_COMMON_ATTR.
+        For elegant style you can add 'arrowhead=open, arrowtail=open, arrowsize=0.5'.
```

Comment:
the `DOT_COMMON_ATTR` should be a link in the form `\ref cfg_dot_common_attr "DOT_COMMON_ATTR"`
the `DOT_EDGE_ATTR` should be like `\c DOT_EDGE_ATTR`

Reply:
Fixed